Realm/RLMRealm.mm

```diff
@@ -766,6 +766,32 @@ + (NSUInteger)schemaVersionAtPath:(NSString *)realmPath encryptionKey:(NSData *)
     return RLMRealmSchemaVersion(realm);
 }
 
++ (BOOL)migrationRequiredAtPath:(NSString *)realmPath encryptionKey:(NSData *)key error:(NSError **)outError {
+    RLMRealm *realm = RLMGetThreadLocalCachedRealmForPath(realmPath);
+    if (realm) {
+        // we've already successfully opened a realm, so a migration isn't necessary.
+        return NO;
+    } else {
+        NSError *error;
+        key = validatedKey(key) ?: keyForPath(realmPath);
+        realm = [[RLMRealm alloc] initWithPath:realmPath key:key readOnly:YES inMemory:NO dynamic:YES error:&error];
+        if (error) {
+            RLMSetErrorOrThrow(error, outError);
+            // the realm could not be created, which means it cannot be migrated.
+            return NO;
+        }
+    }
+    for (RLMObjectSchema *objectSchema in [[RLMSchema sharedSchema] objectSchema]) {
+        RLMObjectSchema *tableSchema = [RLMObjectSchema schemaFromTableForClassName:objectSchema.className realm:realm];
```

Comment:
It doesn't. Should just continue if `tableSchema` is nil (since creating missing tables doesn't require a migration).